.github/workflows/test.yml

```diff
@@ -1,5 +1,5 @@
 name: test
-on: [push]
+on: [push, pull_request]
```

Comment:
Github Actions checks were not running for PRs without this: https://github.com/heimweh/go-pagerduty/pull/96/checks